### DISCUSSION TITLE
Quorum only node

### DISCUSSION THREAD

**Discussion by ahdzesq** [rabbitmq/cluster-operator#1790] [Q&A] [answered]
Is there any way to setup a node that only participates in quorum decisions?

I'm planning to deploy a cluster with 5 nodes

2 nodes Zone-A
2 nodes Zone-B
1 node Zone-C (quorum-only node)

thanks in advance for your recommendations!

**Comment by kjnilsson** [comment, marked as answer]
No. The quorum is based on the replicated log so without it you cannot vote
and retain data safety.

*Karl Nilsson*
